MoreLinq/RandomSubset.cs

```diff
@@ -23,6 +23,45 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
+        /// </summary>
+        /// <typeparam name="T">The type of source sequence elements.</typeparam>
+        /// <param name="sequence">
+        /// The sequence from which to return random elements.</param>
+        /// <returns>
+        /// A random sequence of elements in random order from the original
+        /// sequence.
+        /// </returns>
+        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> sequence)
+        {
+            return Shuffle(sequence, new Random());
+        }
+
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
+        /// </summary>
+        /// <typeparam name="T">The type of source sequence elements.</typeparam>
+        /// <param name="sequence">
+        /// The sequence from which to return random elements.</param>
+        /// <param name="rand">
+        /// A random generator used as part of the selection algorithm.</param>
+        /// <returns>
+        /// A random sequence of elements in random order from the original
+        /// sequence.
+        /// </returns>
+        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> sequence, Random rand)
```

Comment:
Add blank line for clarity.

Also, even though this builds on top of `RandomSubset` is really one thing and the same, can we move this (along with its overloads) to another source file called `Shuffle.cs`?